src/tutorial/microbe_editor/MicrobeEditorTutorialGUI.tscn

```diff
@@ -181,6 +184,114 @@ ShowCloseButton = false
+ShowDelay = 0.7
+LabelPath = NodePath("../StaySmall/ScrollContainer/VBoxContainer/CustomRichTextLabel")
+
+[node name="ScrollContainer" type="ScrollContainer" parent="FlagellumPlacementTutorial"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+rect_min_size = Vector2( 230, 230 )
+scroll_horizontal_enabled = false
+
+[node name="VBoxContainer" type="VBoxContainer" parent="FlagellumPlacementTutorial/ScrollContainer"]
+margin_right = 375.0
+margin_bottom = 385.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+
+[node name="CustomRichTextLabel" parent="FlagellumPlacementTutorial/ScrollContainer/VBoxContainer" instance=ExtResource( 5 )]
+margin_right = 375.0
+margin_bottom = 24.0
+custom_fonts/normal_font = ExtResource( 4 )
+ExtendedBbcode = ""
+
+[node name="Control" type="Control" parent="FlagellumPlacementTutorial/ScrollContainer/VBoxContainer"]
+margin_top = 28.0
+margin_right = 375.0
+margin_bottom = 349.0
+size_flags_vertical = 3
+
+[node name="Button" type="Button" parent="FlagellumPlacementTutorial/ScrollContainer/VBoxContainer"]
+margin_left = 132.0
+margin_top = 353.0
+margin_right = 242.0
+margin_bottom = 385.0
+rect_min_size = Vector2( 110, 0 )
+size_flags_horizontal = 4
+custom_fonts/font = ExtResource( 6 )
+text = "CLOSE"
+
+[node name="ChemoreceptorPlacementTutorial" parent="." instance=ExtResource( 9 )]
+anchor_left = 0.5
+anchor_top = 0.5
+anchor_right = 0.5
+anchor_bottom = 0.5
+margin_left = -182.5
+margin_top = -167.5
+margin_right = 192.5
+margin_bottom = 217.5
+rect_min_size = Vector2( 375, 385 )
+script = ExtResource( 7 )
+__meta__ = {
+"_editor_description_": ""
+}
+WindowTitle = "TUTORIAL"
+Description = "TUTORIAL_MICROBE_EDITOR_CHEMORECEPTOR"
+DescriptionForController = ""
+ShowDelay = 0.7
+LabelPath = NodePath("../StaySmall/ScrollContainer/VBoxContainer/CustomRichTextLabel")
+
+[node name="ScrollContainer" type="ScrollContainer" parent="ChemoreceptorPlacementTutorial"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+rect_min_size = Vector2( 230, 230 )
+scroll_horizontal_enabled = false
+
+[node name="VBoxContainer" type="VBoxContainer" parent="ChemoreceptorPlacementTutorial/ScrollContainer"]
+margin_right = 375.0
+margin_bottom = 385.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+
+[node name="CustomRichTextLabel" parent="ChemoreceptorPlacementTutorial/ScrollContainer/VBoxContainer" instance=ExtResource( 5 )]
+margin_right = 375.0
+margin_bottom = 24.0
+custom_fonts/normal_font = ExtResource( 4 )
+ExtendedBbcode = ""
+
+[node name="Control" type="Control" parent="ChemoreceptorPlacementTutorial/ScrollContainer/VBoxContainer"]
+margin_top = 28.0
+margin_right = 375.0
+margin_bottom = 349.0
+size_flags_vertical = 3
+
+[node name="Button" type="Button" parent="ChemoreceptorPlacementTutorial/ScrollContainer/VBoxContainer"]
+margin_left = 132.0
+margin_top = 353.0
+margin_right = 242.0
+margin_bottom = 385.0
+rect_min_size = Vector2( 110, 0 )
+size_flags_horizontal = 4
+custom_fonts/font = ExtResource( 6 )
+text = "CLOSE"
```

Comment:
Could this tutorial be made to close when the player starts the organelle modify process? Then this close button can be removed and the tutorial flow will be nicer ensuring the tutorial disappears once the player does what the tutorial wants them to do.